frontend/src/core/api/comment.js

```diff
@@ -25,4 +25,14 @@ export default class CommentAPI extends APIBase {
 
         return this.fetch('/add-comment/', 'POST', payload, headers);
     }
+
+    /**
+     * Get all users from server.
+     */
+    async getUsers(): Promise<Object> {
+        const headers = new Headers();
+        headers.append('X-Requested_With', `XMLHttpRequest`);
+
+        return await this.fetch('get-users', 'GET', null, headers);
+    }
```

Comment:
I went back and forth on this and ultimately decided to put it in `comment.js` because it is ALL users as opposed to data specific to an individual user and comments is the only place it is used.  

With that said, it did still feel a bit strange and anti-intuitive to have it there; so, I'll move it to `user.js`